openslides_backend/action/actions/mediafile/add_custom_mimetypes.py

```diff
@@ -0,0 +1,13 @@
+import mimetypes
+
+
+def add_mimetypes() -> None:
+    """
+    for mimetypes we use the mimetypes of the current python version.
+    If we miss some, we could add them here.
+    """
+    mimetypes.add_type("font/ttf", ".ttf")
+    mimetypes.add_type("font/otf", ".otf")
+    mimetypes.add_type("application/x-7z-compressed", ".7z")
+    mimetypes.add_type("application/vnd.oasis.opendocument.text", ".odt")
+    mimetypes.add_type("application/vnd.oasis.opendocument.spreadsheet", ".ods")
```

Comment:
The backend works with filenames and their extension to guess the mimetypes. Therefore we don't need for instance 'font/sfnt', because the extension is `.otf`. The other way round a filename `xyz.otf` always get the mimetype `font/ttf`. The mimetype/extension combination `woff2` will be recognized in mimetypes guesing, but I appended it as only needed type in the allowed_mimetypes list for font setting.